build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
         jcenter()
```

Comment:
Think travis is not working because `minSdkVersion` has not been moved here after being deleted from AndroidManifest.xml